src/stores/deck.js

```diff
@@ -23,23 +20,218 @@ function shuffleDeck(deck) {
   return array;
 }
 
+function mapItemToCard(item) {
+  return {
+    id: `shop_${item.id}`,
+    rank: '★',
+    suit: '☆',
+    name: item.name,
+    suitName: item.name,
+  };
+}
+
+function defaultsFor(id) {
+  const name = id === 'standard' ? 'Standard Deck' : 'Custom Deck';
+  const desc =
+    id === 'standard'
+      ? 'A standard 52 card deck. So it would seem...'
+      : 'Build a custom deck from owned cards.';
+  return { name, desc };
+}
+
 const useDeckStore = create((set, get) => ({
+  baseDeck: generateDeck(),
   deck: [],
+  ownedShopCards: [],
+  activeDeckId: 'standard',
```

Comment:
We only have one deck for now. Let’s just solve the simplest version of the problem instead of trying to scale it instantly. We only need to be able to write cards to the deck in this store. Once we want to add more decks we will probably need a separate store for that. This store should just track cards in an array.